pkg/utils/consts/consts.go

```diff
@@ -40,3 +40,7 @@ const (
 
 	NodePluginInstallTimeout = 60 * time.Second
 )
+
+var (
```

Comment:
```suggestion
const (
```

Reply:
It's a trick here. We will get the address of this variable. Thus, it should be a var instead of const. lol